packages/@ember/helper/index.ts

```diff
@@ -314,3 +314,4 @@
 
 export { setHelperManager, helperCapabilities as capabilities } from '@glimmer/manager';
 export { invokeHelper, hash, array, concat, get, fn } from '@glimmer/runtime';
+export { default as uniqId} from "../-internals/glimmer/lib/helpers/unique-id";
```

Comment:
```suggestion
export { default as uniqueId } from '../-internals/glimmer/lib/helpers/unique-id';
```

Reply:
Acknowledged! Also adding a test and putting this in draft for now!